numpy/ma/tests/test_core.py

```diff
@@ -3944,7 +3944,7 @@ def test_axis_methods_nomask(self):
 
     def test_mean_as_ndarray(self):
         # Test if mean behaves like ndarray
-        for dtype in [np.float16, np.float32, np.float64, np.float128]:
+        for dtype in [np.float16, np.float32, np.float64]:
```

Comment:
You can use `np.longdouble` here, which is defined correctly (it can be effectively identical to `np.double`, but that is fine).

Reply:
Thank you very much, I changed float64 to double and float128 to longdouble.